### PR TITLE
Fixed permission issue

### PR DESCRIPTION
**Pull request by dbooth-boston**
Fixed wrong permissions on reg test files, and improved the reg test drivers.

### CODE REVIEW

File: RDF-Pipeline/t/accept-test.perl [hunk 1]
```diff
@@ -9,6 +9,9 @@
 
 use strict;
 
+# Enable files created in /var/www to have the right group permissions:
+umask 002;
+
```
Comment (glenna-m) [inline RDF-Pipeline/t/accept-test.perl:14]: Shall we add a comment on what 002 represents?

**Comment by dbooth-boston** [comment]
not needed I think

**Comment by glenna-m** [comment]
OK - LGTM

**Comment by glenna-m** [comment]
LGTM

**Pull request merged**